test/geos_capi/tc_misc.rb

```diff
@@ -50,6 +20,42 @@ def setup
         end
 
 
+        def test_marshal_dump_with_geos
+          @factory = ::RGeo::Geos.factory(
+            :srid => 4326,
+            :wkt_generator => :geos,
+            :wkb_generator => :geos,
+            :wkt_parser => :geos,
+            :wkb_parser => :geos
+          )
+
+          dump = nil
+          assert_nothing_raised { dump = @factory.marshal_dump }
+          assert_equal({}, dump['wktg'])
+          assert_equal({}, dump['wkbg'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.